src/app-layout/visual-refresh/layout.scss

```diff
@@ -156,7 +156,7 @@ explicitly set in script.
   &.has-sticky-notifications.has-sticky-background {
     #{custom-props.$offsetTopWithNotifications}: calc(
       var(#{custom-props.$offsetTop}) + var(#{custom-props.$notificationsGap}) +
-        var(#{custom-props.$notificationsHeight}) + #{awsui.$space-scaled-s}
+        var(#{custom-props.$notificationsHeight}) + #{awsui.$space-static-s}
```

Comment:
Are these 12px to account for the bottom padding of `.header-controls`, which uses `awsui.$space-container-header-vertical`? If so, what about using that same variable to make it harder for them to get out of sync?

Reply:
No: the the bottom padding of `.header-controls` grows the whole dark header, pushing the table header accordingly.

This value represents the `margin-bottom` of the notifications, when the header is sticky:

<img width="1403" alt="Screenshot 2023-07-05 at 10 03 17" src="https://github.com/cloudscape-design/components/assets/29541876/01f79d42-0ee4-45ea-8246-feefe6ee7f0e">


But thinking about it, I believe the margins should change in compact mode, therefore I'll change the fix to change the `height` of the dark header instead!